openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIComponentMapper.java

```diff
@@ -841,6 +840,7 @@ private void extractedConstraintAnnotation(MetadataNode metadata,
         NodeList<AnnotationNode> annotations = metadata.annotations();
         annotations.stream()
                 .filter(this::isConstraintAnnotation)
+                .filter(this::isNotConstraintDate)
                 .filter(annotation -> annotation.annotValue().isPresent())
                 .forEach(annotation -> {
                     MappingConstructorExpressionNode annotationValue = annotation.annotValue().get();
```

Comment:
Relevant to the this comment [1] 
```suggestion
                .filter(annotation -> annotation.annotValue().isPresent())
                .forEach(annotation -> {
                    if (isDateConstraint(annotation)) {
                        DiagnosticMessages errorMsg = DiagnosticMessages.OAS_CONVERTOR_120;
                        IncompatibleResourceDiagnostic error = new IncompatibleResourceDiagnostic(errorMsg,
                annotation.location(), annotation.toString());
                        diagnostics.add(error);
                        return;
                    }
                    MappingConstructorExpressionNode annotationValue = annotation.annotValue().get();
```

[1] - https://github.com/ballerina-platform/openapi-tools/pull/1559/files#r1363662026

Reply:
Resolved!